Various style/compatibility fixes (#36)
endless-sky/endless-sky.github.io: _posts/2017-01-17-unstable-release-0.9.5.md

The way depreciation is implemented is that every ship and outfit loses value each day until it is 360 days old, at which point it is worth 25% of its original value. If you sell it, it stays "in stock" and can be bought back at the depreciated price until you leave the planet. Captured ships and plundered outfits always start out fully depreciated. Depreciation does not in any way affect the performance of a ship or outfit; a brand new ship is no more powerful or fast than a fully depreciated one.

> <img class="centered shadowed" src="/images/depreciation.png" width="500" height="400" alt="A graph showing how the value of ships decreases as they are kept for longer, with 50% of the value being around 80 days.">

Reviewer: ```suggestion
<img class="centered shadowed" src="/images/depreciation.png" width="500" height="400" alt="A graph showing how the value of ships decreases as they are kept for longer, having about 50% of their original value after 80 days">
```

They should either all have periods, or none of them.
